src/Features/Core/Portable/xlf/FeaturesResources.zh-Hans.xlf

```diff
@@ -602,7 +602,7 @@ Make sure to use the "tt" specifier for languages for which it's necessary to ma
       </trans-unit>
       <trans-unit id="Modifying_body_of_member_will_prevent_the_debug_session_from_continuing_due_to_internal_error">
         <source>Modifying the body of '{0}' will prevent the debug session from continuing due to internal error: {1}</source>
-        <target state="needs-review-translation">修改“{0}”的主体将防止调试会话继续，原因是出现内部错误: {1}</target>
+        <target state="translated">修改“{0}”的主体将防止调试会话继续，原因是出现内部错误: {1}</target>
         <note>{1} is a multi-line exception message including a stacktrace. Place it at the end of the message and don’t add any punctation after or around {1}</note>
```

Comment:
Comment is already here

Reply:
Thanks for pointing it out. This is definitely an oversight from the translators. Have made this comment.